IMPLEMENTATION_STATUS.md

```diff
@@ -0,0 +1,588 @@
+
+#### Core
+
+
+--------------------------------------------------------------------------------
+
+## Block `ipfs block`
+
+#### CLI
+
+#### HTTP
+
+| Endpoint                                     | Go Impl       | JS Impl       |
+| -------------------------------------------- | :-----------: | :-----------: |
+| **`GET /api/v0/block/get`**                  | :green_apple: | :green_apple: |
+|     `arg=`                                   | :green_apple: | :green_apple: |
+| **`POST /api/v0/block/put`**                 | :green_apple: | :green_apple: |
+|     `arg=`                                   | :green_apple: | :green_apple: |
+| **`GET /api/v0/block/stat`**                 | :green_apple: | :green_apple: |
+|     `arg=`                                   | :green_apple: | :green_apple: |
+
+#### Core
+
+
+--------------------------------------------------------------------------------
+
+## Bootstrap
+
+#### CLI
+
+#### HTTP
+
+| Endpoint                                     | Go Impl       | JS Impl       |
+| -------------------------------------------- | :-----------: | :-----------: |
+| [`GET /api/v0/bootstrap/add`]                | :green_apple: | :green_apple: |
+| [`GET /api/v0/bootstrap/list`]               | :green_apple: | :green_apple: |
+| [`GET /api/v0/bootstrap/rm`]                 | :green_apple: | :green_apple: |
+
+#### Core
+
+
+--------------------------------------------------------------------------------
+
+## Config
+
+#### CLI
+
+#### HTTP
+
+| Endpoint                                     | Go Impl       | JS Impl       |
+| -------------------------------------------- | :-----------: | :-----------: |
+| [`POST /api/v0/config`]                      | :green_apple: | :chestnut:    |
+| [`POST /api/v0/config/replace`]              | :green_apple: | :chestnut:    |
+| [`GET /api/v0/config/show`]                  | :green_apple: | :chestnut:    |
+| [`POST /api/v0/log/level`]                   | :green_apple: | :chestnut:    |
+| [`GET /api/v0/log/ls`]                       | :green_apple: | :chestnut:    |
+| [`GET /api/v0/log/tail`]                     | :green_apple: | :chestnut:    |
+
+
+#### Core
+
+
+--------------------------------------------------------------------------------
+
+## DAG
+
+#### CLI
+
+#### HTTP
+
+#### Core
+
+--------------------------------------------------------------------------------
+
+## Diagnostics `ipfs diag`
+
+#### CLI
+
+#### HTTP
+
+| Endpoint                                     | Go Impl       | JS Impl       |
+| -------------------------------------------- | :-----------: | :-----------: |
+| [`GET /api/v0/diag/cmds`]                    | :green_apple: | :chestnut:    |
+| [`GET /api/v0/diag/cmds/clear`]              | :green_apple: | :chestnut:    |
+| [`GET /api/v0/diag/cmds/set-time`]           | :green_apple: | :chestnut:    |
+| [`GET /api/v0/net`]                          | :green_apple: | :chestnut:    |
```

Comment:
I went ahead and:
- updated https://github.com/ipfs/http-api-docs
- published the website with a new version of https://github.com/ipfs/website/blob/master/content/docs/api.md
- added a notice tot he old http-api-spec https://github.com/ipfs/http-api-spec#psa---this-specification-is-out-of-date-please-use-api-docs-for-a-up-to-date-reference